INSTALL.rst

```diff
@@ -167,21 +167,54 @@ Media assets
 
 Next, we need to build the assets for the Zenodo application.
 
-To compile Zenodo assets you will need some NPM pacakges and asset building
-tools: NodeJS, SASS, CleanCSS, UglifyJS and RequireJS.
-The easiest way is to install them system-wide, and in the specific versions we
-have pinned. You can do that by executing:
+To compile Zenodo assets we will need to install:
+
+* NodeJS **7.4** and NPM **4.0.5**
+
+* Asset-building dependencies: SASS **3.8.0**, CleanCSS **3.4.19**, UglifyJS **2.7.3** and RequireJS **2.2.0**
+
+If you system packages provide NodeJS and NPM in the versions listed above, you
+can install the asset tools system-wide (with ``sudo``), by executing:
 
 .. code-block:: console
 
    (zenodo)$ sudo ./scripts/setup-npm.sh
 
 Take a look in the script above to see which commands are being run.
-Since those pacakges are installed with ``-g`` flag (system-wide),
-you will need to run the command above with ``sudo``.
+Use of ``sudo`` is required because of the ``-g`` flag for global installation.
```

Comment:
There's also this solution for avoiding `sudo` and installing npm in the user's home directory: https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md

Reply:
I guess to not further complicate this small step in a guide I will leave it as is, since installing npm in home directory is almost the same efforts as using nvm at this point.